scripts/demangle.py

```diff
@@ -6,10 +6,18 @@
 import io
 from logging import error, info, warning
 import cpp_demangle
-import argparse
 from typing import Optional
 from common import *
 from source_code import *
+from enum import IntEnum
+
+
+class SourceCodeStatus(IntEnum):
```

Comment:
Can you put this in `common.py` (or create a new file `const.py`)? I feel like this is equally related to `demangle.py` and `source_code.py`, so it might cause confusion if placed in either file.